_posts/2017-06-27-greenpithumb.md

```diff
@@ -86,18 +64,9 @@ The code for GreenPiThumb is open-source and features:
 
 # Hardware architecture
 
-{% capture fig_img %}
-[![GreenPiThumb wiring diagram](https://raw.githubusercontent.com/JeetShetty/GreenPiThumb/master/doc/greenpithumb_wiring.png)](https://raw.githubusercontent.com/JeetShetty/GreenPiThumb/master/doc/greenpithumb_wiring.png)
-{% endcapture %}
-
-{% capture fig_caption %}
-GreenPiThumb wiring diagram ([downloadable file](https://github.com/JeetShetty/GreenPiThumb/tree/master/doc))
-{% endcapture %}
+{% assign fig_caption = "GreenPiThumb wiring diagram ([downloadable file](https://github.com/JeetShetty/GreenPiThumb/tree/master/doc))" | markdownify | remove: "<p>" | remove: "</p>" %}
 
-<figure>
-  {{ fig_img | markdownify | remove: "<p>" | remove: "</p>" }}
-  <figcaption>{{ fig_caption | markdownify | remove: "<p>" | remove: "</p>" }}</figcaption>
-</figure>
+{% include image.html file="greenpithumb_wiring.png" alt="GreenPiThumb wiring diagram" img_link="true" fig_caption=fig_caption %}
```

Comment:
Can we actually keep this as the [external URL](https://raw.githubusercontent.com/JeetShetty/GreenPiThumb/master/doc/greenpithumb_wiring.png)? I'd prefer to just keep one authoritative copy of the wiring diagram.

Reply:
Absolutely, this has image has been reverted to the original external URL in 4903898